screen-orientation/window-orientation.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>window.orientation API</title>
+<link rel="help" href="https://w3c.github.io/screen-orientation/">
```

Comment:
`orientationchange` event isn't referenced in Screen Orientation API now.  Is https://compat.spec.whatwg.org/ better? This document is https://compat.spec.whatwg.org/#windoworientation-interface.  Or will you add a document for `window.orientation` in Screen Orientation API?

Reply:
I'll add it to screen orientation.